functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SurefireForksConfigurationTest.java

```diff
@@ -81,7 +84,10 @@ private void verifyTestSuiteExecution(String... systemPropertiesPairs){
                 .withSystemProperties(SMART_TESTING_REPORT_ENABLE, "true")
             .configure()
             .run();
+
         // then
+        String projectMavenLog = project.getMavenLog();
+        assertThat(projectMavenLog).contains(SUREFIRE_PROVIDER_LOGS);
```

Comment:
check also maven extension logs. and in this particular case use the smart testing debug mode and check specific strings when the extension is loaded, during the surefire provider phase and again in maven extension when the session is ended